coderd/database/queries/organizations.sql

```diff
@@ -53,3 +53,20 @@ INSERT INTO
 VALUES
 	-- If no organizations exist, and this is the first, make it the default.
 	($1, $2, $3, $4, $5, (SELECT TRUE FROM organizations LIMIT 1) IS NULL) RETURNING *;
+
+-- name: UpdateOrganization :one
+UPDATE
+	organizations
+SET
+	updated_at = $2,
+	name = $3
+WHERE
+	id = $1
+RETURNING *;
```

Comment:
I prefer names, but your call
```suggestion
-- name: UpdateOrganization :one
UPDATE
	organizations
SET
	updated_at = @updated_at,
	name = @name
WHERE
	id = @id
RETURNING *;
```